corehq/apps/userreports/reports/view.py

```diff
@@ -359,9 +362,12 @@ class CustomConfigurableReportDispatcher(ReportDispatcher):
 
     @staticmethod
     def _report_class(domain, config_id):
-        class_path = StaticReportConfiguration.report_class_by_domain_and_id(
-            domain, config_id
-        )
+        try:
```

Comment:
not blocking but this seems a little bit like the wrong place for the error checking - in that it's now coupling the property with an HTTP workflow. it would be better if this check happened right when the view is loaded instead of here.

reasons I don't think it's too important in this case:
- the method is private
- the class is obviously involved in handling views